scripts/generate_client.py

```diff
@@ -170,7 +169,7 @@ def generate_client() -> str:
 
     code = [
         "# GENERATED FROM scripts/gen-client.py -- DO NOT EDIT",
-        f"# Last Modified: {datetime.now()}",
+        "# Last Modified: `git log -1 --pretty=\"format:%ci\" pygls/lsp/client.py`",
```

Comment:
Perhaps it's best to just remove this line entirely?

I only added it as a "hey, look the script works!" kind of thing but never ended up deleting it and it's not like it adds anything since we can get that information from git - as you're command demonstrates.